proxies/nodejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "@devcycle/nodejs-server-sdk": "latest",
+    "@devcycle/nodejs-server-sdk": "*",
```

Comment:
Just curious, what is the purpose of this change?

Reply:
I didn't notice that the package was getting updated when it was set to `latest`, which should be the same thing as setting it to `*`.